ui/.build/src/sass.ts

```diff
@@ -2,124 +2,180 @@ import * as cps from 'node:child_process';
 import * as fs from 'node:fs';
 import * as ps from 'node:process';
 import * as path from 'node:path';
+import clr from 'tinycolor2';
 import { env, colors as c, lines, errorMark } from './main';
-import { buildModules } from './build';
 import { globArray } from './parse';
 
-const partialRe = /(.*)\/_(.*)\.scss$/;
-const importRe = /@import ['"](.*)['"]/g;
-const sassArgs = ['--no-error-css', '--stop-on-error', '--no-color', '--quiet', '--quiet-deps'];
+type ColorMix = { c1: string; c2?: string; op: string; val: number };
```

Comment:
I will probably refactor this as a class and fix a few redundant compilations when creating mix sources, but it's not the highest priority.

The rest of this pull request is more of the same except for dasher where there's a few tiny changes. There's also the readme.md which is targeted more towards graphic designer sorts but it does explain the mixing in a bit more detail.